src/webapp/components/DeliveryTable.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableContainer from "@material-ui/core/TableContainer";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+import Paper from "@material-ui/core/Paper";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles({
+  container: {
+    maxHeight: 440
+  }
+});
+
+const DeliveryTable = ({ rows }) => {
+  const classes = useStyles();
+
+  return (
+    <TableContainer className={classes.container} component={Paper}>
+      <Table aria-label="simple table">
+        <TableHead>
+          <TableRow>
+            <TableCell>Code</TableCell>
+            <TableCell align="right">Cross Street #1</TableCell>
```

Comment:
We should be using `str` translations here as well. See README for details on how to add i18n.

Reply:
These strings in particular might already exist in the webapp namespace!